openmls/src/lib.rs

```diff
@@ -31,7 +31,7 @@ mod codec;
 
 #[cfg(any(feature = "expose-test-vectors", test))]
 #[macro_use]
-pub mod test_util;
+pub mod test_utils;
```

Comment:
Can you change the feature to something like `test-utils`?

Reply:
Done.